src/platform/packages/shared/response-ops/alerts-delete/components/modal.tsx

```diff
@@ -183,6 +186,20 @@ export const AlertDeleteModal = ({
     },
   });
 
+  const { mutate: alertDeleteScheduleSubmit } = useAlertDeleteSchedule({
+    services: { http },
+    onSuccess: () => {
+      notifications.toasts.addSuccess(i18n.ALERT_DELETE_SUCCESS);
+      onClose();
+    },
+    onError: (error: IHttpFetchError<ResponseErrorBody>) => {
+      notifications.toasts.addDanger({
+        title: i18n.ALERT_DELETE_FAILURE,
+        text: error.body?.message || JSON.stringify(error),
```

Comment:
I think we should avoid stringing the whole error in the modal. What about `error.body?.message ?? error.message ?? 'Unknown error'`

Reply:
done in done in [df3981f](https://github.com/elastic/kibana/pull/217226/commits/df3981f4a07a02c9373393de39c5c933e8767e17)